changelog/@unreleased/pr-234.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: fix
+fix:
+  description: Correctly respond 204 on empty optional values
```

Comment:
Given we're introducing a breaking code change, let's be more descriptive about what changed and what users should do to handle it. We'll most likely want to send out an announcement that this is changing and refer back to these release notes which should be descriptive enough that users won't have to look into the code examples.

Reply:
Updated, let me know what you think!